package.json

```
@@ -6,13 +6,14 @@
   "license": "UNLICENSED",
   "scripts": {
     "prepare": "husky install",
-    "prettier:check": "prettier --check \"{,src/**/}*.{md,json,yml,html,js,ts,tsx,css,scss,vue,java}\"",
-    "prettier:format": "prettier --write \"{,src/**/}*.{md,json,yml,html,js,ts,tsx,css,scss,vue,java}\""
+    "prettier:check": "prettier --check \"{,src/**/}*.{md,json,yml,html,js,ts,tsx,css,scss,vue,java,xml}\"",
+    "prettier:format": "prettier --write \"{,src/**/}*.{md,json,yml,html,js,ts,tsx,css,scss,vue,java,xml}\""
   },
   "devDependencies": {
+    "@prettier/plugin-xml": "^1.1.0",
```

I'd prefer to use fixed version, as we have tons of issue in generator-jhipster when using `^`

```suggestion
    "@prettier/plugin-xml": "1.1.0",
```